src/settings.c

```diff
@@ -41,6 +41,14 @@ static void on_sidebar_pos_left_changed(GSettings *settings, gchar *key, gpointe
 }
 
 
+static void on_editor_font_changed(GSettings *settings, gchar *key, gpointer user_data)
+{
```

Comment:
We could do it that way too, there would be one `on_setting_changed` handler and it would "switch" on the `key` parameter. Not sure which way is better.